tests/store/model_registry/test_sqlalchemy_store.py

```diff
@@ -28,1559 +28,1569 @@
 from mlflow.store.model_registry.sqlalchemy_store import SqlAlchemyStore
 from tests.helper_functions import random_str
 
-DB_URI = "sqlite:///"
+# DB_URI = "sqlite:///"
 
 pytestmark = pytest.mark.notrackingurimock
 
 
-class TestSqlAlchemyStoreSqlite(unittest.TestCase):
-    def _get_store(self, db_uri=""):
-        return SqlAlchemyStore(db_uri)
-
-    def _setup_db_uri(self):
-        if _TRACKING_URI_ENV_VAR in os.environ:
-            self.temp_dbfile = None
-            self.db_url = os.getenv(_TRACKING_URI_ENV_VAR)
-        else:
-            fd, self.temp_dbfile = tempfile.mkstemp()
-            # Close handle immediately so that we can remove the file later on in Windows
-            os.close(fd)
-            self.db_url = "{}{}".format(DB_URI, self.temp_dbfile)
-
-    def setUp(self):
-        self._setup_db_uri()
-        self.store = self._get_store(self.db_url)
-
-    def get_store(self):
-        return self.store
-
-    def tearDown(self):
-        if self.temp_dbfile:
-            os.remove(self.temp_dbfile)
-        else:
-            with self.store.ManagedSessionMaker() as session:
-                for model in (
-                    SqlModelVersionTag,
-                    SqlRegisteredModelTag,
-                    SqlModelVersion,
-                    SqlRegisteredModel,
-                ):
-                    session.query(model).delete()
-
-    def _rm_maker(self, name, tags=None, description=None):
-        return self.store.create_registered_model(name, tags, description)
-
-    def _mv_maker(
-        self,
-        name,
-        source="path/to/source",
-        run_id=uuid.uuid4().hex,
-        tags=None,
-        run_link=None,
-        description=None,
-    ):
-        return self.store.create_model_version(
-            name, source, run_id, tags, run_link=run_link, description=description
-        )
+# @pytest.fixture
+# def db_uri():
+#     if _TRACKING_URI_ENV_VAR in os.environ:
+#         return os.getenv(_TRACKING_URI_ENV_VAR)
```

Comment:
This needs to be kept to run tests in this file with different databases

Reply:
In that case, I don't I can use the `tmp_sqlite_uri` fixture from `conftest.py`.

Reply:
I pushed a commit to update the `store` fixture.